docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc

```diff
@@ -564,41 +564,43 @@ EOF
 done
 ----
 +
-<1> If your deployment has a custom DNS Domain, modify the `spec:nodes:[NODE NAME]:hostName` to use fqdn for the node.
-<2> The networks composition must match the source cloud configuration to avoid dataplane connectivity downtime. The ctlplane network must come first. The above commands only retain IP addresses for the hosts on `ctlplane` and `internalapi` networks. Complete it the same way for other networks, or update the resulted files manually.
-<3> Use node sets names, like `openstack-cell1`, `openstack-cell2`. Only create node sets for cells containing compute nodes.
-<4> If TLS Everywhere is enabled, change `spec.tlsEnabled` to `true`.
-<5> If not adopting the telemetry services, omit it from the services list.
-<6> The bridge name and other OVN and Neutron specific values must match the source cloud configuration to avoid dataplane connectivity downtime.
+<1> If your deployment has a custom DNS Domain, specify the FQDN for the node.
+<2> The network composition must match the source cloud configuration to avoid data plane connectivity downtime. The `ctlplane` network must come first. The commands only retain IP addresses for the hosts on the `ctlplane` and `internalapi` networks. Complete it the same way for other networks, or update the resulting files manually.
+//Bohdan: Can you clarify what you mean by "Complete it the same way"? What does "it" refer to? And the same way as what?
```

Comment:
it refers to the marked code block which should be used the same way for other isolated networks